selfdrive/car/honda/interface.py

```diff
@@ -204,6 +204,32 @@ def get_params(candidate, fingerprint):
       ret.longitudinalKpV = [1.2, 0.8, 0.5]
       ret.longitudinalKiBP = [0., 35.]
       ret.longitudinalKiV = [0.18, 0.12]
+    elif candidate == "HONDA ODYSSEY 2018 EX-L":
+      stop_and_go = False
+      ret.m = 4354./2.205 + std_cargo
```

Comment:
these variable names are old. Looks like you need to rebase your branch with latest devel.

Reply:
@rbiasini My branch is based off of @energee 's branch for easier merging of Odyssey port and Pilot Port. Would you rather have me change variable names in this PR or submit new PR with updated variables

Reply:
@vanillagorillaa , up to you. You can either:
- create a PR based off master
- wait on @energee  to rebase his branch and then rebase yours.

Since the 2 PR's should't intersect, maybe 1 is better.

Reply:
I'll rebase first